src/Css.elm

```diff
@@ -13069,3 +13071,58 @@ lineClamp :
     -> Style
 lineClamp (Value val) =
     AppendProperty ("line-clamp:" ++ val)
+
+
+{-| Sets [`mix-blend-mode`](https://css-tricks.com/almanac/properties/m/mix-blend-mode/)
+
+    mixBlendMode normal
+
+    mixBlendMode multiply
+
+    mixBlendMode screen
+
+    mixBlendMode overlay
+
+    mixBlendMode darken
+
+    mixBlendMode lighten
+
+    mixBlendMode colorDodge
+
+    mixBlendMode colorBurn
+
+    mixBlendMode hardLight
+
+    mixBlendMode softLight
+
+    mixBlendMode difference
+
+    mixBlendMode exclusion
+
+    mixBlendMode saturation
```

Comment:
I think it may be a little much to demonstrate examples for all 13 of these. Would you mind choosing two or so?